operator/internal/controller/redpanda/redpanda_controller.go

```diff
@@ -489,6 +489,22 @@ func (r *RedpandaReconciler) reconcileLicense(ctx context.Context, rp *v1alpha2.
 		return err
 	}
 
+	licenseInfo, err := client.GetLicenseInfo(ctx)
+	if err != nil {
+		if internalclient.IsTerminalClientError(err) {
+			apimeta.SetStatusCondition(rp.GetConditions(), metav1.Condition{
```

Comment:
Think it would be good to log the error out here, perhaps at trace or debug? There's similar code to this in V1 that ended up making me break glass because I was unable to see the error being reported.

Reply:
So we set the error message to the status condition `ClusterLicenseValid`. So folks should actually see it when they just do a `kubectl get redpanda ...` in the column I added. Is that enough or are you thinking for like centralized logging purposes it'd still be useful?